conans/test/functional/command/copy_packages_test.py

```diff
@@ -27,8 +27,8 @@ class Pkg(ConanFile):
         packages = os.listdir(pkgdir)
         self.assertEqual(len(packages), 3)
 
-        # Copy just one
-        client.run("copy Hello0/0.1@lasote/stable pepe/stable -p %s" % packages[0])
+        # Copy just one with --package argument
+        client.run("copy Hello0/0.1@lasote/stable:%s pepe/stable" % packages[0])
```

Comment:
I think I would leave one test (for each command) that keeps the old ``-p=ID`` argument, to make sure that we are not breaking existing users.

Reply:
OK, I will take that as an opportunity to refactor the tests a bit.